### PR TITLE
Bump to 1.17.3. Remove redundant directive, solving mime warning on boot

### PR DESCRIPTION
**Pull request by akira28**
Relevant docs: Enables gzipping of responses for the specified MIME types in addition to “text/html”. The special value “*” matches any MIME type (0.8.29). Responses with the “text/html” type are always compressed.

http://nginx.org/en/docs/http/ngx_http_gzip_module.html#gzip_types

### CODE REVIEW

**Review by gustauperez — APPROVED** [review]
Looks ok.

File: rootfs/etc/nginx/conf.d/default.template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 vhost_traffic_status_zone;
 
 gzip on;
-gzip_types text/html text/plain application/json;
+gzip_types text/plain application/json;
```
Comment (rccrdpccl) [inline rootfs/etc/nginx/conf.d/default.template:4]: Is this removed because it's already included somewhere else?

Comment (gustauperez) [reply rootfs/etc/nginx/conf.d/default.template:4]: It's there by default.

Comment (akira28) [reply rootfs/etc/nginx/conf.d/default.template:4]: read the description of the PR :)

Comment (rccrdpccl) [reply rootfs/etc/nginx/conf.d/default.template:4]: Oops, i've missed the description, sorry :)

**Review by rccrdpccl — APPROVED** [review]

**Pull request merged**